### PR TITLE
[LittleLink Custom] new guide for LittleLink Custom

### LINKED ISSUE #1327: [LittleLink Custom] Add a guide for LittleLink Custom
I'm going to write a guide for [LittleLink Custom](https://littlelink-custom.com/), an open source alternative to Linktree and Co.

### PR DESCRIPTION
**Pull request by escuco**
fixes #1327

### CODE REVIEW

File: source/guide_littlelinkcustom.rst
Comment (EV21) [inline source/guide_littlelinkcustom.rst:46]: Hi 👋,
since PHP 7.4 is nearing its EOL, have you tried version 8.0 or 8.1?

Comment (escuco) [reply source/guide_littlelinkcustom.rst]: Hello!

Thank you! Yes, I've tested the installation and functionality with PHP 8.1 and it works fine.

I've updated the guide now.

File: source/guide_littlelinkcustom.rst
Comment (noave) [inline source/guide_littlelinkcustom.rst:13]: the uploaded image is a *.png and the link is thus incorrect

File: source/guide_littlelinkcustom.rst
Comment (noave) [inline source/guide_littlelinkcustom.rst:62]: please show some output, for example:

```
[...]
2022-12-14 18:16:56 (73.3 MB/s) - ‘littlelink-custom.zip’ saved [24926989/24926989]
```

File: source/guide_littlelinkcustom.rst
Comment (noave) [inline source/guide_littlelinkcustom.rst:64]: even so, show some of the last output lines:

```
[...]
extracting: littlelink-custom/version.json  
inflating: littlelink-custom/web.config  
inflating: littlelink-custom/webpack.mix.js  
inflating: README.md
```

File: source/guide_littlelinkcustom.rst
Comment (noave) [inline source/guide_littlelinkcustom.rst:67]: proposing easier way for download->unzip and creating structure:

```
$ cd /var/www/virtual/$USER/ #one level above html
$ wget https://github.com/JulianPrieber/littlelink-custom/releases/latest/download/littlelink-custom.zip
$ unzip littlelink-custom.zip littlelink-custom/* #get only the files from the subfolder
$ rm html/noconten.html; rmdir html/ #remove the default html folder in a safe way
$ rm littlelink-custom.zip
$ mv littlelink-custom html
```

by the way, you dont need to put an empty `[isabell@stardust ~]$` this is just thought to be at the end of a codeblock

File: source/guide_littlelinkcustom.rst
Comment (noave) [inline source/guide_littlelinkcustom.rst:83]: it looks like the setup way has changed in between, you have to choose a DB etc. Please advise before to create DB `isabell_littlelink` (lookup other guides for the corresponding CREATE code) and choose that one in the web installer.

If there is a way to execute the installation not in web but on the terminal that would be a better way for the guide

**Review by noave — CHANGES_REQUESTED** [review]
it looks like the web installer has changed with the current version, you need to set up your account and database, can you update your PR refleting this?

File: source/guide_littlelinkcustom.rst
Comment (escuco) [reply source/guide_littlelinkcustom.rst]: @noave, thank you for the hints!

I've uploaded the missing image file for the logo, changed the database setup process to MySQL and adjusted the installation process to your change request.

Hope the guide is ok now (I've tested it with success on a fresh installation). Please let me know if there is anything else that I can do.

Thanks!

**Comment by escuco** [comment]
Dear @noave and @EV21 ,

is there anything I can do get my pull request merged? It still says "Changes requested", but I don't know how to change this status after that I've edited the pull request according to your suggestions. Do I have to manually trigger something in this pr?

Please let me know, If you're missing something or when I have to set any status in this pull request.

Thank you!

**Comment by EV21** [comment]
Hi @escuco ,

you correctly marked your pull request as ready for review and you also request one of the maintainers (they got notified), so you don't have to do anything but wait.

The fact that there is still a change request is because GitHub expects a change at line 13. - This change request maker can be either removed by changing this line or simply by approval by the reviewer/maintainer.
You can simply ignore this last change request as this is nothing unusual at GitHub or if you want you can rename this Link and the File (littlelinkcustom.svg -> LittleLinkCustom.svg) so there is a change.

The guide is already available as a preview at https://deploy-preview-1328--uberlab.netlify.app/guide_littlelinkcustom/ 👀

I am not part of Uberspace, so I have no power.

**Comment by escuco** [comment]
Hello @EV21 , 

thank you for the explanation. Then I'll wait for the feedback or approval by the maintainer.

**Comment by luto** [comment]
Thank you for the guide and thanks @EV21 for the review! I just tested it and it works great! Sorry about the delay :(

**Pull request merged**
